docs/uninstall.md

```
@@ -0,0 +1,188 @@
+
+If the operator is rolled back to a version without explicit uninstall support:
+- The `uninstall` field is preserved by Kubernetes but ignored by the old operator
+- Packages at `StageUninstall` will be handled by the old version-change logic
+- **Before rolling back**: remove `uninstall` config from all CRs to avoid packages stuck in `apply: true` state
+
+## Troubleshooting
+
+### Package stuck in uninstall
+
+Check the uninstall pod logs:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)